bin/sl-nginx-ctl.js

```diff
@@ -73,35 +82,59 @@ if (parser.optind() !== argv.length) {
   process.exit();
 }
 
-if (controlUri === null) {
-  console.error('Command API endpoint was not specified, try `%s --help`.', $0);
+if (disableApi && !controlUri) {
+  console.error(
+    'Either the control URL or an API endpoint must be provided, ' +
```

Comment:
"Must either connect to a control point (-C) or listen for API requests (-L)"

I like because it says in more human terms what the problem is, the other wording is closer to "must do A or B", and you have to look up A and B to understand why one of those is necessary. Just to elaborate a bit more on why I'm so nitpicky about CLI interface